PocketKit/Sources/PocketKit/Collections/CollectionViewModel.swift

```diff
@@ -347,3 +365,10 @@ extension CollectionViewModel: NSFetchedResultsControllerDelegate {
         }
     }
 }
+
+// TODO: NATIVE COLLECTIONS - Update when working on analytics ticket
+extension CollectionViewModel {
+    func trackerForReportRecommendation() -> Tracker {
+        tracker.childTracker(hosting: .reportDialog)
```

Comment:
I guess this will be updated at a later time, just noticing that we are currently using a `childTracker` in a `Context`, which is now deprecated

Reply:
yeah I kept it similar to what we have existing for the report a recommendation and didn't want to refactor that. I can pass in the normal `tracker` for the saves case and can also refactor that for home